src/data-workspace/category-combo-table.js

```diff
@@ -47,7 +47,7 @@ export const CategoryComboTable = ({
 
     if (
         computedCategoryOptions.length !==
-        categoryCombo.categoryOptionCombos.length
+        (categoryCombo.categoryOptionCombos?.length || 1)
```

Comment:
This is a [magic number](https://eslint.org/docs/rules/no-magic-numbers). My preference would be to avoid that and use a descriptive name for what this value actually stands for (+ maybe a comment, because I'm confused by the `1`; if the variable name is not expressive enough)

Reply:
Yeah I think I will remove this. Some categoryCombos are not retuned with the `categoryOptionCombos`from the API. This should just be attribute-combos so it shouldn't affect this part of the app. But a bug in the API made the default one also not return it, and made the app crash, so this was just a simple fix. I fixed the default-problem another way in `use-metadata.js`

Reply:
With the fix that Lars did last work, could this `1` be removed completely then?